src/thor/bidirectional_astar.cc

```diff
@@ -607,8 +626,11 @@ BidirectionalAStar::GetBestPath(valhalla::Location& origin,
         continue;
       }
 
+      // Prepare the time information before expansion
```

Comment:
Is this a TODO? I don't see any preparation?

Reply:
this must have been me figuring out where i wanted to do this work. in the other time dependent algorithms old implementations you need to do some math right before you call expand. i guess i was preparing to add it here (i frequently put a bunch of comments in where i need to add code before i start adding the code). looks like i added it and then forgot to remove it when i changed my mind about how to implement this